sbin/build.sh

```diff
@@ -505,6 +504,14 @@ configureDebugParameters() {
   fi
 }
 
+configureAlsaLocation() {
+  if [[ ! "${CONFIGURE_ARGS}" =~ "--with-alsa" ]]; then
```

Comment:
Could merge those two if clauses.

Reply:
Right. Could.  In  `configureFreetypeLocation`  and in `configureZlibLocation`  are similar structures, although in first they can not be merged, and in second it would not be nice.
It seemed better to me to keep it like this for readability. Bash conditions can be terrible.

I'm voting to keep it as it is. If you insist, will redo asap.